app/graphql/types/notification_event_type.rb

```diff
@@ -0,0 +1,7 @@
+module Types
+  class NotificationEventType < Types::BaseEnum
+    ::Notification.events.map do |key, value|
+      value key, "Notification triggered when #{value.split('.').first} is #{value.split('.').second}"
+    end
```

Comment:
This will break the naming convention we've been using for GraphQL enum keys, which is `CamelCase`. I'd `.camelcase` the key before using it.

```suggestion
    ::Notification.events.map do |key, value|
      value key.camelcase, "Notification triggered when #{value.split('.').first} is #{value.split('.').second}"
    end
```